### DISCUSSION TITLE
Which SDK version do you use for IOS right now?

### DISCUSSION THREAD

**Discussion by premiumsB** [airsdk/Adobe-Runtime-Support#3510] [General]
Hello

I'm using the version AIR 50.2.4.3 right now and I was wondering which version do you guys use because mine is a bit old right now and I don't want to use one with major bug?

I prefer to use windows to create my build but I can also use a mac for that.

Let me know guys.
Bobby

**Comment by FliplineStudios** [comment]
My last release for iOS a while back was with 50.2.4.4, packaged on MacOS, since packaging on Windows isn't reliable anymore due to the newer iOS changes.  I believe you would at least need to use 50.2.5.1 for the new privacy requirements though (or manually create the new privacy manifests if using a version earlier than 50.2.5.1).  I'm planning some updates soon and will likely use 50.2.5.1 for those, though maybe 51.1 is fairly stable now?

**Reply by premiumsB** [reply]
@FliplineStudios I uploaded an APP with 50.2.4.3 built with windows last time and I didn't have any problem with the new privacy requirement?

Anybody else had this problem or saw a stable version with 51.1 and above?
